web/src/utilities/react-router-utilities.tsx

```diff
@@ -0,0 +1,54 @@
+import { KotsParams } from "@types";
+import React from "react";
+import {
+  match,
+  RouteComponentProps,
+  useHistory,
+  useLocation,
+  useRouteMatch,
+} from "react-router";
+
+// @ts-ignore
+const RouterWrapper = ({ children }) => {
+  const history = useHistory();
+  const location = useLocation();
+  const wrappedMatch = useRouteMatch<KotsParams>();
+
+  return children({ history, location, wrappedMatch });
+};
+
+/**
+ * @deprecated The method should not be used on new components. New components should use the hooks directly.
```

Comment:
this causes the `withRouter` to show up crossed out with this comment in vscode.